.github/workflows/ci-repo.yml

```
@@ -173,28 +173,128 @@
+
+      - name: Set DISPLAY environment variable
+        run: |
+          Xvfb :99 -screen 0 1920x1080x24 &
+          echo "DISPLAY=:99" >> $GITHUB_ENV
+
+      - name: Start D-Bus
+        run: |
+          dbus-launch --exit-with-session &
+
+      - name: Start Dbus
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2155:warning:2:8: Declare and assign separately to avoid masking return values

(shellcheck)